src/include/processor/operator/scan/scan_rel_table_columns.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "processor/operator/filtering_operator.h"
+#include "processor/operator/scan/scan_rel_table.h"
+
+namespace kuzu {
+namespace processor {
+
+class ScanRelTableColumns : public ScanRelTable, public SelVectorOverWriter {
+public:
+    ScanRelTableColumns(table_id_t boundNodeTableID, DirectedRelTableData* tableData,
+        vector<uint32_t> propertyIds, const DataPos& inNodeIDVectorPos,
+        vector<DataPos> outputVectorsPos, unique_ptr<PhysicalOperator> child, uint32_t id,
+        const string& paramsString)
+        : ScanRelTable{inNodeIDVectorPos, std::move(outputVectorsPos),
+              PhysicalOperatorType::SCAN_REL_TABLE_COLUMNS, std::move(child), id, paramsString},
+          tableData{tableData} {
+        scanState = make_unique<RelTableScanState>(
+            boundNodeTableID, std::move(propertyIds), RelTableDataType::COLUMNS);
+    }
+    ~ScanRelTableColumns() override = default;
```

Comment:
Do we need to override the deconstructor of the base class? I don't see we do this in other operators.

Reply:
We used to have these, they are just extra safety net, but not necessary. I removed. them now.